functions/zoom-downloader.py

```diff
@@ -624,7 +673,7 @@ def stream_file_to_s3(self):
             raise
 
         if self.file_extension == "mp4":
-            if not self.valid_mp4_file:
+            if not self.valid_mp4_file():
```

Comment:
So it turns out because "valid_mp4_file" wasn't even being called here, we were not even running ffprobe!!!

Reply:
whoops!